base_changeset/security/groups.xml

```diff
@@ -15,12 +15,11 @@
         </record>
     </data>
     <data noupdate="1">
-        <record id="group_changeset_manager" model="res.groups">
-            <field name="users" eval="[(4, ref('base.user_root'))]" />
-        </record>
         <record id="group_changeset_user" model="res.groups">
-            <field name="implied_ids" eval="[(4, ref('group_changeset_manager'))]" />
```

Comment:
I would keep `<field name="users" eval="[(4, ref('base.user_root'))]" />` for `group_changeset_user` as well

Reply:
with the now correct implied ides and admin added to the manager group, admin also ends up in the user group, so that's fine I think? I do notice however that we have an empty xml node then, so I'm going to remove that

Reply:
you are right, thanks!

Reply:
thanks, followed up on all your input by now